src/components/search-by-message/SearchByMessage.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import InputWithIcon from '~/components/input-with-icon/InputWithIcon'
+import IconsWithCounter from '../icons-with-counter/IconsWithCounter'
```

Comment:
as i understand we importing like this:
`import IconsWithCounter from '~/components/icons-with-counter/IconsWithCounter'`

Reply:
done